lib/sidekiq/cloudwatchmetrics.rb

```diff
@@ -209,6 +219,8 @@ def publish
 
     # Returns busy / concurrency averaged across processes (for scaling)
     private def calculate_utilization(processes)
+      return 0.0 if processes.empty?
+
```

Comment:
This is to resolve the `NaN` error at runtime.

There is currently an open issue for this: https://github.com/sj26/sidekiq-cloudwatchmetrics/issues/16
These changes are based off of the PR mentioned in the issue [comment](https://github.com/sj26/sidekiq-cloudwatchmetrics/issues/16#issuecomment-921061013).